reactDashboard/src/components/RegionDropdown/RegionDropdown.css

```diff
@@ -1,4 +1,23 @@
 .region-menu {
-    height: 250px;
-    overflow-y: scroll;
+  height: 250px;
+  overflow-y: scroll;
+  width: 220px;
+}
+
+.selected-region {
+  margin: 10px 0;
+  width: 220px;
+}
+
+.darkmode .selected-region {
+  color: #121212;
```

Comment:
Part of the confusion here is that lightmode is using standard bootstrap theming, and darkmode is overriding specific instances of buttons, rather than overriding the theming. Next PR should clean that up - lightmode and darkmode should use the same selectors where possible